Data/FE14/DialogueCommands.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "Command": "$HasPermanents",
+    "Args": "",
+    "Hint": "<strong><font color =\"#8b008b\">$HasPermanents</font></strong><br><strong>Effect</strong>: Allow for the player's name or arguments to be used in a conversation. This typically goes at the top of the script before anything else."
+  },
+  {
+    "Command": "$ConversationType",
+    "Args": "",
+    "Hint": "<strong><font color =\"#8b008b\">$ConversationTypeCommand</font></strong>(<strong><font color=\"#0000FF\">type</font></strong>: <strong><font color=\"#1b8b51\">u8</font></strong>)<br> <strong>Effect</strong>: Change the display type used for the conversation. 0 is typically used for battle conversations. 1 is for conversations with full busts like supports and most cutscenes."
+  },
+  {
+    "Command": "$Color",
+    "Args": "",
+    "Hint": ""
+  },
+  {
+    "Command": "$NewSpeaker",
+    "Args": "Character",
+    "Hint": "<strong><font color =\"#8b008b\">$NewSpeaker</font></strong>(<strong><font color=\"#0000FF\">character</font></strong>: <strong><font color=\"#1b8b51\">String</font></strong>)<br> <strong>Effect</strong>: Add a new speaker to the conversation. After this command, you should use the <strong><font color =\"#8b008b\">$Reposition</font></strong> command to place them in the scene. The speaker name is determined by the name value in the <strong><code>\"Portraits / FaceData\"</code></strong> module. If you need to use different portraits with the same name, add a portrait override to <strong><code>\"Modules/ServiceData/FE14PortraitOverrides.json\"</code></strong>."
+  },
+  {
+    "Command": "$Reposition",
```

Comment:
There were some changes to commands with the rewrite, so some pieces of this file may need to be renamed. Reposition was removed entirely in favor of specifying the speaker position in LoadAssets.